enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java

```diff
@@ -381,10 +382,11 @@ public void addSeparatedClasses(List<ClassEntry> obfClasses, List<ClassEntry> de
 				return;
 			}
 
-			ClassEntry deobfEntry = mapper.deobfuscate(entry);
+			TranslateResult<ClassEntry> result = mapper.extendedDeobfuscate(entry);
+			ClassEntry deobfEntry = result.getValue();
 
 			List<ObfuscationTestService> obfService = enigma.getServices().get(ObfuscationTestService.TYPE);
-			boolean obfuscated = deobfEntry.equals(entry);
+			boolean obfuscated = result.isObfuscated() && deobfEntry.equals(entry);
```

Comment:
Is the `deobfEntry.equals(entry)` still needed here?

Reply:
I don't think so, but I think it'd be best if these were cleaned up in another PR where classes marked deobfucated but with their obfuscated name aren't bugged in the Swing UI